src/feedback/views/veranstalter.py

```diff
@@ -256,7 +255,9 @@ def get_template_names(self):
         return [VERANSTALTER_VIEW_TEMPLATES[self.steps.current]]
 
     def done(self, form_list, **kwargs):
-        cleaned_data = self.get_cleaned_basisdaten()
+        cleaned_data = {}
+        if perform_evalution(self) :
+            cleaned_data = self.get_cleaned_basisdaten()
```

Comment:
Is the `if` necessary? I think it is a good thing when lecturers get notified of a successful submission, even if they don't want to evaluate.

Reply:
This `if` was added because of django-formtools update. `get_cleaned_basisdaten` is using `get_cleaned_data_for_step` which uses `get_form` which uses `get_form_list`.

Starting from django-fromtools 2.4, _"WizardView is using get_form_list() instead of directly accessing form_list"_ as in [chaneglog](https://django-formtools.readthedocs.io/en/latest/changelog.html). `get_form_list()` is using functions in `condition_dict` to determine whether to add or not to add steps.

As `perform_evalution` is assigned to `basisdaten` in `condition_dict`, it returns `False`, because we don't want to evaluate, which also causes `basisdaten` not to get added to steps in `get_form_list`.

In `get_form` there is `self.get_form_list()[step]`, which throws Error, as `basisdaten` is not in output of `get_form_list`.

Reply:
Ok, thanks for the explanation! Would you mind adding a short comment in the source code on why this is necessary?

Reply:
I added the comment, I think this is as short as it gets, without leaving anything important out